app/components/syntax-highlighted-diff.ts

```diff
@@ -138,13 +160,8 @@ export default class SyntaxHighlightedDiffComponent extends Component<Signature>
 
   @action
   handleIsDarkModeUpdate(isDarkMode: boolean) {
-    if (isDarkMode !== this.isDarkMode) {
-      // Avoid re-use in same computation bug
-      next(() => {
-        this.isDarkMode = isDarkMode;
-        this.highlightCode.perform();
-      });
```

Comment:
Using `next` is unnecessary and harmful: it _leads_ to computation bugs and future errors in tests.